zuul.d/molecule.yaml

```diff
@@ -1,305 +1,304 @@
-# DO NOT EDIT - generaged using make role_molecule
 - job:
-    name: cifmw-molecule-artifacts
-    parent: cifmw-molecule-base
-    vars:
-      TEST_RUN: artifacts
     files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/artifacts/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
-- job:
-    name: cifmw-molecule-build_openstack_packages
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/polarion/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-polarion
     parent: cifmw-molecule-base
     vars:
-      TEST_RUN: build_openstack_packages
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/build_openstack_packages/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
+      TEST_RUN: polarion
 - job:
-    name: cifmw-molecule-ci_setup
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/os_must_gather/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-os_must_gather
```

Comment:
seems to lack the correct nodeset

Reply:
Updated the parent based on @raukadah info https://github.com/openstack-k8s-operators/ci-framework/pull/401/files#diff-e11625dd628d0cfeeaad59207c1fabdd877fb86c87eeca532b83571e184628dcR20